engines/engine.cpp

```diff
@@ -326,18 +326,16 @@ void initGraphics(int width, int height, const Graphics::PixelFormat *format) {
 #endif
 
 	if (gfxError & OSystem::kTransactionModeSwitchFailed) {
-		Common::String message = _("Could not switch to video mode: '");
-		message += ConfMan.get("gfx_mode");
-		message += "'.";
+		Common::String message;
+		message = Common::String::format(_("Could not switch to video mode '%s'."), ConfMan.get("gfx_mode"));
```

Comment:
This is invalid. `ConfMan.get()` returns a `Common::String`, while variadic functions, such as `format()` can only take POD arguments (some compilers may accept non-POD arguments, but this is not portable).

You can use `ConfMan.get("gfx_mode").c_str()`, which returns a `char *`, to fix the issue.